spec/features/trainer_spec.rb

```diff
@@ -75,6 +79,7 @@
       user.cards.each { |card| card.update_attribute(:review_date,
                                                      Time.now - 3.days) }
       visit trainer_path
+      click_link 'ru'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.